packages/eslint-plugin/configs/react.js

```diff
@@ -30,5 +30,6 @@ module.exports = {
 		'react/prop-types': 'off',
 		'react/react-in-jsx-scope': 'off',
 		'react-hooks/rules-of-hooks': 'error',
+		'react-hooks/exhaustive-deps': 'warn',
```

Comment:
Not sure if it's `'warn'` for the purpose of initial discussion, but we shouldn't want to introduce this to the codebase as a warning. It will be ignored.

Related: #18458